src/core/load-fragment.c

```diff
@@ -2860,7 +2860,15 @@ int config_parse_device_allow(
                 return 0;
         }
 
-        n = strcspn(rvalue, WHITESPACE);
+        int r;
```

Comment:
Please see CODING_STYLE. Please declare variables at the beginning of blocks, not in the middle.